openstack/autoscaling/v1/groups/results.go

```diff
@@ -80,45 +34,7 @@ type Group struct {
 }
 
 type Network struct {
-	ID            string        `json:"id"`
-	IPv6Enable    bool          `json:"ipv6_enable"`
-	IPv6Bandwidth IPv6Bandwidth `json:"ipv6_bandwidth"`
-}
-
-type IPv6Bandwidth struct {
-	ID string `json:"id"`
-}
-
-type SecurityGroup struct {
-	ID string `json:"id"`
-}
-
-type LBaaSListener struct {
-	ListenerID   string `json:"listener_id"`
-	PoolID       string `json:"pool_id"`
-	ProtocolPort int    `json:"protocol_port"`
-	Weight       int    `json:"weight"`
-}
-
-type GroupPage struct {
```

Comment:
Does this pager really help? Do we need to keep it?

Reply:
This is a `SinglePageBase`, pagination without pagination